runtime/storage/cache/basic.go

```diff
@@ -0,0 +1,563 @@
+package cache
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+)
+
+// NewStringKeyspace creates a keyspace that stores string values in the given cluster.
+//
+// The type parameter K specifies the key type, which can either be a
+// named struct type or a basic type (string, int, etc).
+func NewStringKeyspace[K any](cluster *Cluster, cfg KeyspaceConfig) *StringKeyspace[K] {
+	fromRedis := func(val string) (string, error) { return val, nil }
+	toRedis := func(val string) (any, error) { return val, nil }
+
+	return &StringKeyspace[K]{
+		&basicKeyspace[K, string]{
+			newClient[K, string](cluster, cfg, fromRedis, toRedis),
+		},
+	}
+}
+
+// StringKeyspace represents a set of cache keys that hold string values.
+type StringKeyspace[K any] struct {
+	*basicKeyspace[K, string]
+}
+
+// Get gets the value stored at key.
+// If the key does not exist, it returns an error matching Miss.
+func (s *StringKeyspace[K]) Get(ctx context.Context, key K) (string, error) {
+	return s.basicKeyspace.Get(ctx, key)
+}
+
+// Set updates the value stored at key to val.
+// If the key does not exist, it returns an error matching Miss.
```

Comment:
is this true? I assumes this would add a key/val

Reply:
Whoops :) good catch!